client/topbar.jsx

```diff
@@ -0,0 +1,30 @@
+const React = require('react'),
+      Avatar = require('./avatar.jsx'),
+      Link = require('react-router-dom').Link,
+      i18n = require('i18next-client');
+
+const TopBar = React.createClass({
+    getInitialState() {
+        return {
+            username: null
+        };
+    },
+    onLogin(username) {
+        this.setState({username});
+    },
+    render() {
+        return (
+            <div className='dropdown topbar'>
+                <button className='btn btn-outline-primary dropdown-toggle' type='button' data-toggle='dropdown'>
+                    <Avatar username={this.state.username} />
+                </button>
+                <ul className='dropdown-menu dropdown-menu-left'>
+                    <li><Link to='/settings'>{i18n.t('topbarSet.settings')}</Link></li>
+                    <li><Link to='/exit'>{i18n.t('topbarSet.exit')}</Link></li>
```

Comment:
This doesn't do anything. It should log me out. (refreshing the page currently should be sufficient)